tests/test_networks.py

```diff
@@ -805,7 +805,7 @@ def test_address_valid(value, name, email):
         ('\u0020@example.com', None),
         ('\u001f@example.com', None),
         ('"@example.com', None),
-        ('\"@example.com', None),
+        ('"@example.com', None),
```

Comment:
Another odd change

Reply:
Ruff's formatter removes unnecessary escape sequences and picks the quote style that leads to the fewest escape (with the motivation that escapes reduce readability). Also see https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings

Reply:
I believe the backslash is unnecessary here -- that is, the string is identical with and without the backslash:

```python
>>> '\"@example.com'
'"@example.com'
```

Reply:
That's fair. I suppose we should just remove the test case then if it's identical to the one directly above.

Reply:
(Sorry, I think Micha and I just hit a race condition here, didn't mean to overload you with my follow-up comment!)

Reply:
No worries at all. I removed the redundant test case and am marking this as resolved :)